test/CadlRanchProjects/type/union/src/Generated/Models/ModelWithSimpleUnionProperty.cs

```diff
@@ -32,6 +33,17 @@ public ModelWithSimpleUnionProperty(BinaryData simpleUnion)
         /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
         /// </para>
         /// <para>
+        /// <remarks>
+        /// Supported types:
+        /// <list type="bullet">
+        /// <item>
+        /// <description><see cref="int"/></description>
+        /// </item>
+        /// <item>
+        /// <description><see cref="IList{T}"/> Where <c>T</c> is of type <c>int</c></description>
```

Comment:
we can make the `type <c>int</c>` a cref as well since it could be a complex type.